### PR TITLE
feat(esl-panel-group): add `refresh-state` attribute

### PR DESCRIPTION
**Pull request by NastaLeo**
Part of [#1132](https://github.com/exadel-inc/esl/issues/1132)

### CODE REVIEW

File: src/modules/esl-panel-group/core/esl-panel-group.shape.ts [hunk 1]
```diff
@@ -21,6 +21,9 @@ export interface ESLPanelGroupTagShape extends ESLBaseElementShape<ESLPanelGroup
   /** Define a list of comma-separated "modes" to disable collapse/expand animation (for both Group and Panel animations)*/
   'no-animate'?: string;
 
+  /** Define active panels behaviour in case of mode changing. Supported values: `last|initial`*/
+  'refresh-state': string;
```
Comment (yadamskaya) [inline src/modules/esl-panel-group/core/esl-panel-group.shape.ts:25]: Can it be not mandatory attribute with `initial` value by default?

**Review by yadamskaya — APPROVED** [review]

File: src/modules/esl-panel-group/core/esl-panel-group.shape.ts (hunk 1 above)
Comment (NastaLeo) [reply src/modules/esl-panel-group/core/esl-panel-group.shape.ts:25]: Exactly. Thanks

**Pull request merged**